conda/recipes/cudf/meta.yaml

```diff
@@ -35,7 +35,7 @@ requirements:
     - protobuf
     - python
     - typing_extensions
-    - pandas >=1.0,<1.3.0dev0
+    - pandas >=1.0,<=1.2.3
```

Comment:
Given it's only a test failure related to a Pandas bug I think we can unpin this?

Reply:
Done 👍